PhysicsTools/PatAlgos/python/producersLayer1/jetUpdater_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-patJetsUpdated = cms.EDProducer("PATJetUpdater",
+updatedPatJets = cms.EDProducer("PATJetUpdater",
```

Comment:
In https://github.com/cms-sw/cmssw/pull/12863#discussion-diff-48741766 it was mentioned that this is a transient collection, supposedly not visible to typical analysis use cases.